src/wmnds/components/message/summary/_summary.njk

```diff
@@ -1,52 +1,34 @@
-{% macro wmndsMsgSummary(params) %}
-{# Set vars and defaults #}
+{% macro wmndsMsgSummary(params) %} {# Set vars and defaults #} {# Success #} {% if params.type === "success" %} {% set
+iconName = 'general-success' %} {# Success #} {% elseif params.type === "success-fill" %} {% set iconName =
+'general-success' %} {# Success #} {% elseif params.type === "success-fill-no-icon" %} {% set iconName = '' %} {#
+Warning #} {% elseif params.type === "warning" %} {% set iconName = 'general-warning-circle' %} {# Error #} {% elseif
+params.type === "error" %} {% set iconName = 'general-warning-triangle' %} {# Info (default) #} {% else %} {% set
+iconName = 'general-info' %} {% endif %} {# Set classes #} {% set classes = " " + params.classes if params.classes %} {%
+from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
 
-{# Success #}
-{% if params.type === "success" %}
-  {% set iconName = 'general-success' %}
-{# Success #}
-{% elseif params.type === "success-fill" %}
-  {% set iconName = 'general-success' %}
-{# Success #}
-{% elseif params.type === "success-fill-no-icon" %}
-  {% set iconName = '' %}
-{# Warning #}
-{% elseif params.type === "warning" %}
-  {% set iconName = 'general-warning-circle' %}
-{# Error #}
-{% elseif params.type === "error" %}
-  {% set iconName = 'general-warning-triangle' %}
-{# Info (default) #}
-{% else %}
-  {% set iconName = 'general-info' %}
-{% endif %}
-
-{# Set classes #}
-{% set classes = " " + params.classes if params.classes %}
-
-{% from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
-
-<div class="wmnds-msg-summary wmnds-msg-summary--{{ params.type }} {{classes}}">
+<div class="wmnds-msg-summary wmnds-msg-summary--{{ params.type }} {{ classes }}">
   {%- if params.dismissable %}
-  <button class="wmnds-msg-summary__close wmnds-link">Close 
+  <button class="wmnds-msg-summary__close wmnds-link">
+    Close
     <svg><use xlink:href="#wmnds-general-cross" href="#wmnds-general-cross"></use></svg>
   </button>
   {%- endif %}
-  <div class="wmnds-msg-summary__header">
-    {# Get SVG icon and pass class #}
-    {%- if iconName !== "" -%}
+  <div class="wmnds-msg-summary__header wmnds-grid">
```

Comment:
We shouldn't be using the wmnds-grid on components. It should only be used within patterns.